tests/runtime/usdt

```diff
@@ -114,19 +114,25 @@ NAME usdt probes - attach to probes by wildcard file with child
 RUN {{BPFTRACE}} -e 'usdt:./testprogs/usdt_test*::* { printf("here\n" ); exit(); }' -c ./testprogs/usdt_test
 EXPECT Attached 3 probes
 
-NAME usdt probes - attach to probe on multiple files by wildcard
-PROG usdt:./testprogs/usdt*::* { printf("here\n" ); exit(); }
-EXPECT Attached 49 probes
+# TODO(Jiawei Zhao): re-enable this test when libbpf can handle SIB usdt argument spec correctly
+# NAME usdt probes - attach to probe on multiple files by wildcard
+# PROG usdt:./testprogs/usdt*::* { printf("here\n" ); exit(); }
+# EXPECT Attached 49 probes
 
 NAME usdt probes - attach to probe on multiple providers by wildcard and pid
 RUN {{BPFTRACE}} -e 'usdt:::*probe2 { printf("here\n" ); exit(); }' -p {{BEFORE_PID}}
 EXPECT Attached 2 probes
 BEFORE ./testprogs/usdt_test
 
+
+# When uprobe_multi is supported, a single ebpf program can be attached to multiple
+# identical usdt probes, so we only need to attach to 3 different usdt probes.
+# For systems without uprobe_multi support, we need to manually attach an ebpf
+# program to each probe location, resulting in 6 probes being attached.
 NAME usdt probes - attach to multiple probes with different number of locations by wildcard
 PROG usdt:./testprogs/usdt_multiple_locations:tracetest:testprobe* { printf("here\n" ); exit(); }
 BEFORE ./testprogs/usdt_multiple_locations
-EXPECT Attached 6 probes
+EXPECT_REGEX Attached (3|6) probes
```

Comment:
What do you think about having an additional test that utilizes the `REQUIRES_FEATURE uprobe_multi` directive ?